react-native-gutenberg-bridge/android/src/main/java/org/wordpress/mobile/WPAndroidGlue/WPAndroidGlueCode.java

```diff
@@ -430,12 +436,23 @@ public void onCreateView(Context initContext,
         }
         initialProps.putBundle(PROP_NAME_CAPABILITIES, capabilities);
 
+        Serializable colors = editorTheme != null ? editorTheme.getSerializable(PROP_NAME_COLORS) : null;
+        if (colors != null) {
+            initialProps.putSerializable(PROP_NAME_COLORS, colors);
+        }
+
+        Serializable gradients = editorTheme != null ? editorTheme.getSerializable(PROP_NAME_GRADIENTS) : null;
+        if (gradients != null) {
+            initialProps.putSerializable(PROP_NAME_GRADIENTS, gradients);
+        }
+
```

Comment:
I'm wondering if we would have more props that are from the theme. If yes i think it would be worth to implement a method like `addThemeProp(initialProps, propName)` to not duplicate the code since it looks pretty similar

Reply:
That's a good idea. I guess that would partially depend on the other items that are added. For now, though this seems to be the limit of added fields with ongoing discussions on if we'll theme the background or not. I think if we add more here we might want to pursue that idea more.